ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/ThreadNet/Network.hs

```diff
@@ -394,9 +391,6 @@ runThreadNetwork ThreadNetworkArgs
   where
     _ = keepRedundantConstraint (Proxy @(Show (LedgerView (BlockProtocol blk))))
 
-    epochInfo :: EpochInfo m
-    epochInfo = fixedSizeEpochInfo epochSize
-
```

Comment:
@nfrisby you're working on testing hard forks, in which the epoch size will no longer be static. Do you think there will be an oracular `EpochInfo`? If so, wouldn't it live here, in a central location?